### PR TITLE
Fix broken images and formatting in emails

### PR DESCRIPTION
**Pull request by beyondtheteal**
contributes to #75
* fixed broken image links by changing to inline attachments
* fixed a formatting problem due to the size of the logo by adding a CSS style for max-width of 800, per standard email guidelines
* Add ability to preview emails in browser
* Work remaining: Change to inline attachments for remaining emails in thing_mailer.rb

### CODE REVIEW

**Comment by beyondtheteal** [comment]
Deployment test works for first email.

File: app/views/thing_mailer/first_adoption_confirmation.html.erb [hunk 1]
```diff
@@ -65,7 +66,7 @@
   <table width="100%" border="0" cellspacing="0" cellpadding="0">
     <tr>
       <td style="text-align: center;">
-        <%= image_tag image_url("adoptadrain-event.png"), :alt => "adopt-a-drain event", :title => "adopt-a-drain event" %>
+        <%= image_tag attachments['adoptadrain-event.png'].url, :alt => "adopt-a-drain event", :title => "adopt-a-drain event" %>
```
Comment (Wilfongjt) [inline app/views/thing_mailer/first_adoption_confirmation.html.erb]: How do we test this? I don't see a test in the code. We would have caught this earlier had there been one.

Comment (beyondtheteal) [reply app/views/thing_mailer/first_adoption_confirmation.html.erb]: In my next commit there is a test for this!

**Comment by beyondtheteal** [comment]
fixes #105

**Comment by jacebrowning** [comment]
@beyondtheteal Is this still WIP or is it ready for merge after review?

**Comment by beyondtheteal** [comment]
Ready for Merge after review. Emails still need changes/updates but those should be covered by other Issue #s.

**Comment by beyondtheteal** [comment]
@BrandonKlotz it shows you approved the changes but then moved the status back to in progress?

**Comment by BrandonKlotz** [comment]
@beyondtheteal Looks like James made those changes after the review? Maybe he was updating stories in Waffle. I don't know that there is a semantic process for labels yet but would be good to define.
screen shot 2019-02-25 at 1 39 27 pm

@Wilfongjt thoughts on a process with labels? Maybe something to add to the docs?

**Comment by Wilfongjt** [comment]
Bravo

**Pull request merged**